app/mailers/app_mailer.rb

```diff
@@ -8,4 +8,10 @@ def send_forgot_password(user)
     @user = user
     mail to: user.email, from: "info@myflix.com", subject: "Please reset your passowrd"
   end
+
+  def send_invitation_email(invitation)
+    @invitation = invitation
+    mail to: invitation.recipient_email, from: "info@myflix.com", subject: "Invitation to join Myflix"
```

Comment:
You can also use `default from: "info@myflix.com"`, like `before_action` in controllers.